index.bs

```diff
@@ -115,13 +128,13 @@ ISSUE: Should this specification also define a structural brand stricter than 'm
 </pre>
 
 The syntax and semantics of the <dfn>AV1 Item Configuration Property</dfn> are identical to those of the [=AV1CodecConfigurationBox=] defined in [[!AV1-ISOBMFF]], with the following constraints:
-<ul>
-<li>[=Sequence Header OBUs=] SHOULD not be present in the [=AV1 Item Configuration Property=].</li>
-<li>The values of the fields in the [=AV1CodecConfigurationBox=] SHALL match those of the [=Sequence Header OBU=] in the item data and in the [=AV1CodecConfigurationBox=], if any.</li>
-<li>[=Metadata OBUs=], if present, SHALL match the values given in other item properties, such as the PixelInformationProperty or ColourInformationBox.</li>
-</ul>
-This property SHALL be marked as essential.
-</p>
+
+  - [=Sequence Header OBUs=] should not be present in the [=AV1 Item Configuration Property=].
+  - The values of the fields in the [=AV1CodecConfigurationBox=] shall match those of the [=Sequence Header OBU=] in the [=AV1 Image Item Data=].
+  - [=Metadata OBUs=], if present, shall match the values given in other item properties, such as the PixelInformationProperty or ColourInformationBox.
+  - If a [=Sequence Header OBU=] is present in the [=AV1CodecConfigurationBox=], it shall match the  [=Sequence Header OBU=] in the [=AV1 Image Item Data=].
```

Comment:
Done. Also made sure that we refer to "AV1ConfigurationBox" when we mean that, rather than "AV1 Item Configuration Property" which was creating a kind of cyclical reference.